src/basic/terminal-util.c

```diff
@@ -1310,6 +1310,15 @@ int vt_restore(int fd) {
         return q;
 }
 
+int vt_release_display(int fd) {
+        assert(fd >= 0);
```

Comment:
a comment what this does would be greatly appreicated. This is seriously underdocumented in the kernel man pages. 

Maybe something like this would suffice: `/* Tells the kernel that it's OK to switch away now */`

Reply:
ok